com.unity.renderstreaming/Tests/Runtime/RenderStreamingInternalTest.cs

```diff
@@ -327,7 +329,7 @@ public IEnumerator OnAddReceiverPrivateMode()
             var transceiver = target2.AddTrack(connectionId, track);
             Assert.That(transceiver, Is.Not.Null);
 
-            yield return new WaitUntil(() => isAddReceiver1);
+            yield return new WaitUntil(() => isAddReceiver1 && isGotAnswer2);
```

Comment:
If not wait got answer when send answer, next test got error.
`SendAnswer` Task not cancel event if test is ended.